tests/test_DGEList.py

```diff
@@ -74,16 +73,16 @@ def test_library_size():
         dge_list.library_size,
         np.array(
             [
-                90895095,
-                82461005,
-                55676791,
-                111027083,
-                65854416,
-                91305546,
-                95585464,
-                96313896,
-                80069980,
-                52772642,
+                63579607,
+                58531933,
+                39138521,
+                78565885,
+                48667119,
+                62799917,
+                66032107,
+                66194776,
+                55085875,
+                37760315,
             ]
         ),
     )
```

Comment:
Inconsistent np.array structure

Reply:
This is formatting from our automated code formatter, so nothing to do, unfortunately.